lib/parser.js

```diff
@@ -128,7 +131,7 @@ module.exports = function (content, filename, needMap) {
           map.addMapping({
             source: filenameWithHash,
             original: {
-              line: index + 1 + lineOffset,
+              line: index + 1,
```

Comment:
it seems after this change, `lineOffset` is only needed for the padding calculation, so it should be declared directly in the `else` block above.